.github/actions/setup-k3d/action.yaml

```diff
@@ -72,30 +72,52 @@ runs:
       shell: bash
       run: |
         k3d version
-    - name: Setup k3d options
+    - name: Create k8s cluster
       shell: bash
-      id: k3d_options
+      id: start_k3d
       run: |
-        if [ "${AGENTS}" != 0 ]; then
-          OPTIONS="${OPTIONS} --agents ${AGENTS}"
+        if [ -z "${K3D_CLUSTER_NAME}" ]; then
+          K3D_CLUSTER_NAME="vald-cluster"
+        fi
+        if [ -z "${K3D_NODES}" ]; then
+          K3D_NODES=5
```

Comment:
I think `K3D_CLUSTER_NAME` and  `K3D_NODES ` are no empty case because there is a default value in the input settings.

```suggestion
```
```suggestion
```